SmalltalkSources/SBE-Extract.package/SBEWorldHelper.class/instance/setUp.st

```diff
@@ -0,0 +1,16 @@
+running
+setUp
+
+	world := (PasteUpMorph newWorldForProject: nil)
+		extent: self worldSize;
+		viewBox: (0 @ 0 extent: self worldSize);
+		yourself.
+	
+	(world instVarNamed: #worldState)
+		instVarNamed: #canvas
+		put: (Form extent: self worldSize depth: 32) getCanvas. 
+	
+	hand := PHTestHandMorph new.
```

Comment:
@tom95 Not sure why this works on CI. The PHTestHandMorph class is missing I think. Is it important?

Reply:
Hm. Should've copied all of this from the start. I added it to master now.

Reply:
When you want to try it, the fastest way is to adapt the call in `SBExtractImages new runCi` to the path of your repo.